client/wc-api/constants.js

```diff
@@ -22,5 +22,5 @@ export const QUERY_DEFAULTS = {
 	pageSize: 25,
 	period: 'month',
 	compare: 'previous_year',
-	noteTypes: 'info,warning,marketing',
+	noteTypes: 'info,warning,marketing,survey',
```

Comment:
Could we use the consts from WC_Admin_Note?

Reply:
Sorry, I couldn't follow this suggestion, do you think we should load this client constant from the API?

Reply:
Ah, yeah... oops, sorry for making this suggestion. No need to load the constant from the API! Missed that this was client-side JS vs the server-side WC_Admin_Note PHP! 😊